README.md

```diff
@@ -184,6 +188,8 @@ The installation of plugins has a slightly more involved getting-started process
 
 ## Super User Account
 
+If you are testing the application locally, you will need to create a super user below in order to authenticate to the application, even if the credential files are already created from the previous steps.
```

Comment:
Can you clarify what you mean by "even if the credential files are created from previous step"? I believe the super user will be automatically created upon initial startup if the appropriate environment variables are set.

Reply:
Hi @jdrew82 ,

It is probably a user education or user error issue on my part then. From what I can tell the environment variables are set, from updating the creds.env:
```
(nautobot-docker-compose-py3.10) nautobot-docker-compose$ invoke cli
Running docker compose command "ps --services --filter status=running"
Running docker compose command "exec nautobot bash"
nautobot@ba2e8f2e492b:~$ echo $NAUTOBOT_CREATE_SUPERUSER
True
nautobot@ba2e8f2e492b:~$ echo $NAUTOBOT_SUPERUSER_NAME
admin
nautobot@ba2e8f2e492b:~$ echo $NAUTOBOT_SUPERUSER_PASSWORD
admin
nautobot@ba2e8f2e492b:~$ exit
exit
(nautobot-docker-compose-py3.10) nautobot-docker-compose$ docker container ls
CONTAINER ID   IMAGE                                    COMMAND                  CREATED         STATUS                   PORTS                              NAMES
ec1ce64c3b57   yourrepo/nautobot-docker-compose:local   "sh -c 'nautobot-ser…"   9 minutes ago   Up 9 minutes (healthy)   8080/tcp, 8443/tcp                 nautobot-docker-compose-celery_worker-1
ba2e8f2e492b   yourrepo/nautobot-docker-compose:local   "/docker-entrypoint.…"   9 minutes ago   Up 9 minutes (healthy)   0.0.0.0:8080->8080/tcp, 8443/tcp   nautobot-docker-compose-nautobot-1
c7e1c41a9ad0   redis:6-alpine                           "docker-entrypoint.s…"   9 minutes ago   Up 9 minutes             6379/tcp                           nautobot-docker-compose-redis-1
c752cb76df2f   postgres:13-alpine                       "docker-entrypoint.s…"   9 minutes ago   Up 9 minutes (healthy)   5432/tcp                           nautobot-docker-compose-db-1
(nautobot-docker-compose-py3.10) nautobot-docker-compose$
```

But when I try to authenticate into the web UI with admin / admin, I also tried nautobot/nautobot, I get "Please enter a correct username and password. Note that both fields may be case-sensitive." I added that statement because the only way I could get authentication to work was with the manual invoke createsuperuser command. 

I can remove this portion of the update to the README though, and just focus on the flow update.